src/driver/amdxdna/amdxdna_gem.h

```diff
@@ -33,6 +33,7 @@ struct amdxdna_mem {
 	u32				nr_pages;
 	struct list_head		umap_list;
 	bool				map_invalid;
+	bool				cma;
```

Comment:
Why do you need this flag? I think you are using global xdna->cma? This seems redundant.